cirq-google/cirq_google/api/v2/program.proto

```diff
@@ -431,11 +431,31 @@ message ArgMapping {
   repeated ArgEntry entries = 1;
 }
 
+message FunctionInterpolationData {
+  // If the function encoded has dimension `d`, then |x| = d |y|. Where |x| the length of `x`.
+  repeated float x = 1 [packed = true];  // The independent variable.
```

Comment:
IIUC, these fields are meant to evoke `y = f(x)`?

Rather than calling a field `x` and adding a comment to document that it's "the independent variable", WDYT of calling the field something like `independent_variables` to be self-documenting? Note that the suggestion is pluralized to follow the style guidance for repeated fields: https://protobuf.dev/programming-guides/style/#repeated-fields

Reply:
renamed